e2e/network_test.go

```diff
@@ -0,0 +1,307 @@
+package e2e
+
+import (
+	"fmt"
+	"strings"
+	"testing"
+	"time"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	utilrand "k8s.io/apimachinery/pkg/util/rand"
+	"k8s.io/client-go/pkg/api"
+	"k8s.io/client-go/pkg/api/v1"
+	"k8s.io/client-go/pkg/apis/extensions/v1beta1"
+)
+
+func TestNetwork(t *testing.T) {
```

Comment:
@abhinavdahiya A simple flag is good enough, we have something similar for etcd scale test: https://github.com/kubernetes-incubator/bootkube/blob/master/e2e/main_test.go#L32